apps/compreview-todo/todo-backend/Containerfile

```diff
@@ -0,0 +1,26 @@
+# https://github.com/sclorg/s2i-nodejs-container
+FROM registry.access.redhat.com/ubi8/nodejs-14
```

Comment:
We should switch to an internal image. Not something that we have to fix now. We can go GA and fix this afterwards.

Also, I would upgrade the image to Node.js 16 if possible, because the Nexus NPM dependencies have cached by using Node.js 16.
This is just a recommendation. Maybe upgrading to node 16 might break the app

Reply:
This is ok for images we build containers from to populate `redhattraining` , on the other hand there is an issue for the S2I use case of the `apps` folder...